lib/app_web/templates/layout/app.html.eex

```diff
@@ -9,6 +9,7 @@
 
     <title>Github Backup - dwyl</title>
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.9.1/css/tachyons.min.css"/>
+    <link href="https://use.fontawesome.com/releases/v5.0.8/css/all.css" rel="stylesheet">
```

Comment:
we could _consider_ having a _subset_ of Font Awesome to reduce download size (_especially on mobile_).
or we could use "latest" in case people have already downloaded Font Awesome previously.
_However_ I feel this may be "premature optimisation". 
Let's come back to this as _soon_ as one of our _users_ "_complains_" about "slow" load time on Mobile. deal? 👍

Reply:
could we create our own icon? https://github.com/dwyl/hq/issues/435

Reply:
@SimonLab totally! @Cleop will be able to create custom icons. 🎉 
_especially_ if we write a "_how-to_" including image optimisation. 😉